core/src/banking_stage/transaction_scheduler/receive_and_buffer.rs

```diff
@@ -434,7 +434,7 @@ impl TransactionViewReceiveAndBuffer {
                         &mut error_counters,
                     ) {
                         *result = Err(err);
-                        num_dropped_on_status_age_checks += 1;
+                        num_dropped_on_transaction_checks += 1;
```

Comment:
this is count of drop due to fee payer check, the counter `num_dropped_on_receive_transaction_checks` is bit unclear for it. I might have understand counters wrong tho